src/tlapm_args.ml

```diff
@@ -134,140 +107,254 @@ let quote_if_needed s =
+  Arg.(value & opt (some int) None & info ["threads"] ~docv:"N" ~doc)
+
+let method_t =
+  let doc = "Set default method to $(docv) (try --method=help)." in
+  Arg.(value & opt (some string) None & info ["method"] ~docv:"METH" ~doc)
+
+let solver_t =
+  let doc = "Set SMT solver to $(docv)." in
+  Arg.(value & opt (some string) None & info ["solver"] ~docv:"SOLVER" ~doc)
+
+let smt_logic_t =
+  let doc = "Set SMT logic to $(docv)." in
+  Arg.(value & opt (some string) None & info ["smt-logic"] ~docv:"LOGIC" ~doc)
+
+let fast_isabelle_t =
+  let doc = "(Windows-only) Launch Isabelle with fast shortcut." in
+  Arg.(value & flag & info ["fast-isabelle"] ~doc)
+
+let stretch_t =
+  let doc = "Multiply all timeouts by $(docv)." in
+  Arg.(value & opt float 1.0 & info ["stretch"] ~docv:"MULTIPLIER" ~doc)
+
+let noflatten_t =
+  let doc = "Do not flatten obligations." in
+  Arg.(value & flag & info ["noflatten"] ~doc)
+
+let nonormal_t =
+  let doc = "Do not normalize obligations before printing." in
+  Arg.(value & flag & info ["nonormal"] ~doc)
+
+let debug_t =
+  let doc = "Enable/disable debugging flags." in
+  Arg.(value & opt_all string [] & info ["debug"] ~docv:"[-]flag" ~doc)
+
+let toolbox_t =
+  let doc = "Toolbox mode." in
+  Arg.(value & opt (some (pair int int)) None & info ["toolbox"] ~doc)
+
+let toolbox_vsn_t =
+  let doc = "Toolbox protocol version." in
+  Arg.(value & opt int 1 & info ["toolbox-vsn"] ~docv:"1|2" ~doc)
+
+let line_t =
+  let doc = "Line to prove." in
+  Arg.(value & opt (some int) None & info ["line"] ~docv:"LINENO" ~doc)
+
+let wait_t =
+  let doc = "Wait for $(docv) before printing obligations in progress." in
+  Arg.(value & opt int 3 & info ["wait"] ~docv:"TIME" ~doc)
+
+let stdin_t =
+  let doc = "Read the tla file from stdin instead of file system. \
+             Only applies if single tla file is provided as input." in
+  Arg.(value & flag & info ["stdin"] ~doc)
+
+let prefer_stdlib_t =
+  let doc = "Prefer build-in standard modules if the module search path \
+             contains files with the same names as modules in stdlib." in
+  Arg.(value & flag & info ["prefer-stdlib"] ~doc)
+
+let noproving_t =
+  let doc = "Do not prove, report fingerprinted results only." in
+  Arg.(value & flag & info ["noproving"] ~doc)
+
+let printallobs_t =
+  let doc = "Print obligations in all toolbox messages." in
+  Arg.(value & flag & info ["printallobs"] ~doc)
+
+let safefp_t =
+  let doc = "Check tlapm, zenon, Isabelle versions for fingerprints." in
+  Arg.(value & flag & info ["safefp"] ~doc)
+
+let nofp_t =
+  let doc = "Do not use existing fingerprints, \
+             but overwrite any preexisting fingerprints associated \
+             with the current proof obligations, \
+             using the newly computed fingerprints and results" in
+  Arg.(value & flag & info ["nofp"] ~doc)
+
+let nofpl_t =
+  let doc = "Disable fingerprint use between given lines." in
```

Comment:
Same issue with help printer just printing `VAL` instead of anything indicating input format

Reply:
OK.